src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy

```diff
@@ -231,37 +249,57 @@ class DevTask extends AbstractFeatureTask {
+        this.containerRunOpts = containerRunOpts;
+    }
+
+    private String dockerRunOpts;
+    @Option(option = 'dockerRunOpts', description = 'Alias for containerRunOpts')
     void setDockerRunOpts(String dockerRunOpts) {
-        this.dockerRunOpts = dockerRunOpts;
+        if (containerRunOpts == null) {
+            setContainerRunOpts(dockerRunOpts)
+        }
     }
 
-    private int dockerBuildTimeout;
+    private int containerBuildTimeout;
 
-    @Option(option = 'dockerBuildTimeout', description = 'Specifies the amount of time to wait (in seconds) for the completion of the Docker operation to build the image.')
-    void setDockerBuildTimeout(String inputValue) {
+    @Option(option = 'containerBuildTimeout', description = 'Specifies the amount of time to wait (in seconds) for the completion of the container operation to build the image.')
+    void setContainerBuildTimeout(String inputValue) {
         try {
-            this.dockerBuildTimeout = Integer.valueOf(inputValue);
+            this.containerBuildTimeout = Integer.valueOf(inputValue);
         } catch (NumberFormatException e) {
-            logger.error(String.format("Unexpected value: %s for dev mode option dockerBuildTimeout. dockerBuildTimeout should be a valid integer.", inputValue));
+            logger.error(String.format("Unexpected value: %s for dev mode option containerBuildTimeout. containerBuildTimeout should be a valid integer.", inputValue));
             throw e;
         }
     }
 
+    private int dockerBuildTimeout;
+    @Option(option = 'dockerBuildTimeout', description = 'Alias for containerBuildTimeout')
+    void setDockerBuildTimeout(String inputValue) {
+        setContainerBuildTimeout(inputValue)
+    }
+
     private Boolean skipDefaultPorts;
 
-    @Option(option = 'skipDefaultPorts', description = 'If true, the default Docker port mappings are skipped in the docker run command.')
+    @Option(option = 'skipDefaultPorts', description = 'If true, the default container port mappings are skipped in the container run command.')
     void setSkipDefaultPorts(boolean skipDefaultPorts) {
         this.skipDefaultPorts = skipDefaultPorts;
     }
 
-    private Boolean keepTempDockerfile;
+    private Boolean keepTempContainerfile;
 
-    @Option(option = 'keepTempDockerfile', description = 'If true, preserve the temporary Dockerfile used to build the container.')
+    @Option(option = 'keepTempContainerfile', description = 'If true, preserve the temporary Containerfile/Dockerfile used to build the container.')
+    void setKeepTempContainerfile(boolean keepTempContainerfile) {
+        this.keepTempContainerfile = keepTempContainerfile;
+    }
+
+    private Boolean keepTempDockerfile;
+    @Option(option = 'keepTempDockerfile', description = 'Alias for keepTempContainerfile')
     void setKeepTempDockerfile(boolean keepTempDockerfile) {
-        this.keepTempDockerfile = keepTempDockerfile;
+        setKeepTempContainerfile(keepTempDockerfile)
```

Comment:
Why doesn't keepTempContainerFile take precedence here like you did for the other docker attributes?